packages/core/forms/src/components/fields/FieldMultiselect.vue

```diff
@@ -1,41 +1,89 @@
 <template>
   <KMultiselect
+    :id="getFieldID(schema)"
+    v-model="inputValue"
     :aria-labelledby="getLabelId(schema)"
+    :class="schema.fieldClasses"
     data-testid="field-multiselect"
+    :disabled="disabled || undefined"
     :items="items"
+    :kpop-attributes="{ 'data-testid': `${getFieldID(schema)}-items` }"
     :label-attributes="{ info: schema.help }"
-    :model-value="value"
+    :name="schema.inputName"
     :placeholder="schema.placeholder"
     :required="schema.required || undefined"
     width="100%"
-    @update:model-value="onUpdate"
   />
 </template>
 
-<script>
-import abstractField from './abstractField'
+<script lang="ts" setup>
+import { computed, toRefs, type PropType } from 'vue'
+import type { MultiselectItem } from '@kong/kongponents'
+import composables from '../../composables'
 
-export default {
-  mixins: [abstractField],
+const props = defineProps({
+  disabled: {
+    type: Boolean,
+    default: false,
+  },
+  formOptions: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => undefined,
+  },
+  model: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => undefined,
+  },
+  schema: {
+    type: Object as PropType<Record<string, any>>,
+    required: true,
+  },
+  vfg: {
+    type: Object,
+    required: true,
+  },
+  /**
+   * TODO: stronger type
+   * TODO: pass this down to KInput error and errorMessage
+   */
+  errors: {
+    type: Array,
+    default: () => [],
+  },
+  hint: {
+    type: String,
+    default: '',
+  },
+})
```

Comment:
Do we still prefer value syntax instead of type syntax for `defineProps`?

Reply:
I'm not sure there is a preference? The rest of the files in there are using `defineProps` so that's what I went with.